modules/geo-layers/src/tile-layer/tile-2d-header.js

```diff
@@ -32,13 +36,27 @@ export default class Tile2DHeader {
     return result;
   }
 
+  async _loadData(getTileData) {
+    const {x, y, z, bbox, layerId} = this;
+
+    // Construct unique identifier for request scheduler handle
+    const id = `${layerId}-${x}-${y}-${z}`;
```

Comment:
I think it would be better not to expose layer id to the header, it should not know anything about who uses it. 

I believe that you could just use the tileheader instance itself as the handle instead of a string.